content/en/docs/contributing/_index.md

```diff
@@ -6,7 +6,13 @@ weight: 200
 cSpell:ignore: prepopulated spacewhite
 ---
 
-You can open an issue about OpenTelemetry documentation, or contribute a change
+This document is related to contributing do documentation.
+If you are looking on how to contribute to the OpenTelemetry project in general see the
+[OpenTelemetry Contributor Guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)
```

Comment:
could we also include a wording to indicate that every language implementation, collector, conventions repo has their own specific contributing guides as well?

Reply:
Is there also any place that aggregates all contributing guides?

Reply:
Added the wording